code/modules/events/solar_storm.dm

```diff
@@ -31,6 +31,8 @@
 /datum/event/solar_storm/proc/radiate()
 	// Note: Too complicated to be worth trying to use the radiation system for this.  Its only in space anyway, so we make an exception in this case.
 	for(var/mob/living/L in GLOB.living_mob_list_)
+		if(istype(L.loc, /mob/living/exosuit)) //Todo, generalize this further like I did for rads if other things need to block or mitigate heat - CrimsonShrike
```

Comment:
This is the kind of pattern that helped make old mech code absolute shit to deal with. An `ATOM_FLAG_SHIELD_CONTENTS` set on mechs would make more sense.